keras/preprocessing/image.py

```diff
@@ -337,6 +337,60 @@ def list_pictures(directory, ext='jpg|jpeg|bmp|png'):
             if re.match(r'([\w]+\.(?:' + ext + '))', f)]
 
 
+def hsv_augmentation(x, hue_shift,
+                     saturation_scale, saturation_shift,
+                     value_scale, value_shift, dim_ordering='default'):
+    """Function to perform HSV data augmentation by shifting and rescaling the HSV color representations of a batch of images
+       using values drawn from uniform distributions:
+           H += U(-hue_shift, +hue_shift)
+           S *= U(1/(1+saturation_scale), 1+saturation_scale)
+           S += U(-saturation_shift, +saturation_shift)
+           V *= U(1/(1+value_scale), 1+value_scale)
+           V += U(-value_shift, value_shift)
```

Comment:
This function does not have a properly formatted dosctring (see other dosctrings for reference)